packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -1,6 +1,6 @@
 import { inspect } from 'util';
 import { ObjectId, EJSON } from 'bson';
-import { promises as fs } from 'fs';
+import { promises as fs, rmdirSync } from 'fs';
```

Comment:
Is there a specific reason that this method needs to be synchronous?

Reply:
cleanup() in index.ts is synchronous and I need to call it there.

Reply:
See https://github.com/mongodb-js/compass/blob/a58edee9ed9bc7d522a2cd6ccdccc45e2407cb67/packages/compass-e2e-tests/index.ts#L50-L74